tsmarty2c.php

```diff
@@ -220,7 +264,7 @@ function do_dir($outfile, $dir) {
 
 define('PROGRAM', basename(array_shift($argv)));
 define('TMPDIR', sys_get_temp_dir());
-$opt = getopt('o:');
+$opt = getopt('d:o:');
```

Comment:
Make -d an optional param (d::) to support empty/default domain with -d or custom domain with -d=domain